src/lib/sort-rules.js

```diff
@@ -1,9 +1,27 @@
 'use strict'
 
 function getSortedRules(rules) {
-  return rules.sort(function sort(a, b) {
-    return a > b ? 1 : -1
-  })
+  return Array.isArray(rules) ? getSortedRulesArray(rules) : transformIntoSortedRulesArray(rules)
+}
+
+function getSortedRulesArray(rules) {
+  return rules.sort(sortAlphabetically)
+}
+
+function transformIntoSortedRulesArray(rules) {
+  var sortedRules = []
+
+  Object.keys(rules)
+    .sort(sortAlphabetically)
+    .forEach(function map(ruleName) {
+      sortedRules.push(ruleName, rules[ruleName].config1, rules[ruleName].config2)
```

Comment:
I am confused 😕 

Why is it looking different than 

[`{'foo-rule': {config1: [2, 'foo'], config2: [2, 'bar']}`](https://github.com/sarbbottam/eslint-find-rules/pull/50/files#diff-6cd1268db522cc061bcb74dfe44fdf03R8)